Twig/Node/TimelineActionThemeNode.php

```diff
@@ -16,7 +16,7 @@ public function compile(\Twig_Compiler $compiler)
     {
         $compiler
             ->addDebugInfo($this)
-            ->write('echo $this->env->getExtension(\'timeline_render\')->setTheme(')
+            ->write('echo $this->env->getExtension(\'Spy\TimelineBundle\Twig\Extension\TimelineExtension\')->setTheme(')
             ->subcompile($this->getNode('action'))
```

Comment:
Please don't use fcqn here

Reply:
So what do you suggest? using `timeline_render` generates a deprecation.

Reply:
You can use TimelineExtension::class, then `use`